scanner/django.go

```diff
@@ -41,6 +44,69 @@ func configureDjango(sourceDir string, config *ScannerConfig) (*SourceInfo, erro
 
 	vars := make(map[string]interface{})
 
+	var pythonSupported string = "3.7" // https://devguide.python.org/versions/#supported-versions
+	var pythonVersion string = "3.10" // Keep the default version "3.10" (hardcoded on the Dockerfile)
+	var pythonVersionOutput string = "Python 3.10.0"
+	var pythonVersionFound bool = false
+
+	cmd := exec.Command("python3", "-V")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		cmd := exec.Command("python", "-V")
+		out, err := cmd.CombinedOutput()
+		if err == nil {
+		    pythonVersionOutput = string(out)
+		}
+	} else {
+	    pythonVersionOutput = string(out)
+	}
+
+    re := regexp.MustCompile(`Python ([0-9]+\.[0-9]+)\.[0-9]`)
+    match := re.FindStringSubmatch(pythonVersionOutput)
+
+    if len(match) > 1 {
+        // necessary to split and compare the major/minor values because "3.11" is HIGHER than "3.6"
+        // e.g. "3.11" -> "3", "11"
+        version := strings.Split(match[1], ".")
+
+        major, parseMajorErr := strconv.Atoi(version[0])
```

Comment:
Does Python follow SemVer? flyctl uses https://github.com/blang/semver already and maybe you could.

Reply:
@kzys Nice tip! ! I'll check the repo and see if we could use it, it'd make things much easier! 😃 Thanks!